lua/lspconfig/health.lua

```diff
@@ -1,12 +1,14 @@
+local M = {}
+local health = require('vim.health')
+
 local api, fn = vim.api, vim.fn
-local windows = require 'lspconfig.ui.windows'
 local util = require 'lspconfig.util'
 
 local error_messages = {
-  cmd_not_found = 'Unable to find executable. Please check your path and ensure the server is installed',
-  no_filetype_defined = 'No filetypes defined, Please define filetypes in setup()',
+  cmd_not_found = 'Unable to find executable. Check your $PATH and ensure the server is installed.',
+  no_filetype_defined = 'No filetypes defined. Define filetypes in setup().',
   root_dir_not_found = 'Not found.',
-  async_root_dir_function = 'Asynchronous root_dir functions are not supported in :LspInfo',
+  async_root_dir_function = 'Asynchronous root_dir functions are not supported by `:checkhealth lspconfig`',
```

Comment:
I need help with confirming that these messages still work correctly and are displayed nicely. Probably they should be converted to `health.error()` calls.